runtime/src/main/java/org/corfudb/runtime/view/LayoutBuilder.java

```diff
@@ -21,6 +21,8 @@
 @Slf4j
 public class LayoutBuilder {
 
+    public static final int CHAIN_REPLICATION_MIN_NUM_NODES = 2;
```

Comment:
missing doc. This needs to come from some configuration parameter and shouldn't be part of the LayoutBuilder class. (as a static parameter).

Reply:
Added more documentation. 

@medhavidhawan already made this comment, it will be addressed in a different path that will introduce a server options object.

Reply:
Just move it as a parameter to removeFromStripe for now.

Reply:
Done.

Reply:
Done.